tlf.1.in

```diff
@@ -1445,12 +1445,20 @@ updated by WWV or WCY messages.
 .
 .TP
 .B CHANGE_RST
-If set in logcfg.dat, PgUp and PgDown will change RST instead of CW speed if
-field is not empty.
+If set in logcfg.dat, PgUp and PgDown will change his/my RST instead of 
```

Comment:
his/my -> received/sent

Reply:
(in multiple places, just to be clear on the direction)

Reply:
I tested the feature and read the manual, also found this note as Zoli: the his/my should be replaced by received/sent.

Reply:
Zoli, it is the other way round in the code. his_rst is the rapport I give to him, my_rst is what I got.

But I agree that we should use receive/sent in the man page.